octoprint_camera/__init__.py

```diff
@@ -553,7 +556,7 @@ def saveInitialCalibrationMarkers(self):
             json_data = request.get_json()
         except BadRequest:
             return make_response("Malformed JSON body in request", 400)
-        if not all(k in json_data.keys() for k in ["newCorners", "newMarkers"]):
+        if not all(k in list(json_data.keys()) for k in ["newCorners", "newMarkers"]):
```

Comment:
I think this list() isn't necessary. If I remember correctly 2to3 was adding it because .keys() was returning a list before and an another iterable now. But in this case both are fine :)